src/ProjectService.Mappers/RequestsMappers/Interfaces/ICreateRoleRequestMapper.cs

```diff
@@ -0,0 +1,9 @@
+﻿using LT.DigitalOffice.ProjectService.Mappers.Interfaces;
+using LT.DigitalOffice.ProjectService.Models.Db;
+using LT.DigitalOffice.ProjectService.Models.Dto.Requests;
+
+
+namespace LT.DigitalOffice.ProjectService.Mappers.RequestsMappers.Interfaces
+{
+    public interface ICreateRoleRequestMapper : IMapper<DbRole, CreateRoleRequest>, IMapper<CreateRoleRequest, DbRole> { }
```

Comment:
The line goes beyond the recommended limit of 110 characters, so you can transfer interfaces to new lines (see wiki)